### PR TITLE
Add FFT processor

### PR DESCRIPTION
**Pull request by JensGM**
The FFT processor perform FFTs and inverse FFTs on 1- or
2-dimensional datasets

### CODE REVIEW

File: tests/test_camille.py [hunk 1]
```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env python
 import unittest
 import pandas as pd
+import numpy as np
+from scipy import fftpack as fftp
```
Comment (JensGM) [inline tests/test_camille.py]: This is not used

File: camille/processors/fft.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    'ifft': fftp.ifft,
+    'fft2': fftp.fft2,
+    'ifft2': fftp.ifft2,
+    'fftn': fftp.fftn,
+    'ifftn': fftp.ifftn,
+}
+
+def process(df, **kwargs):
+    options = ('dimensions', 'inverse')
+    if not all([key in options for key in kwargs.keys()]):
```
Comment (jokva) [inline camille/processors/fft.py]: `all(key in option for key in kwargs.keys())`

File: camille/processors/fft.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        raise ValueError(
+            'Unknown argument(s) {}'.format(kwargs.keys() - options))
```
Comment (jokva) [inline camille/processors/fft.py]: `set()` around keys and options to make it more py2-3 robust

File: camille/processors/fft.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+    dimensions = kwargs.get('dimensions', __defaults['dimensions'])
+    inverse = kwargs.get('inverse', __defaults['inverse'])
+
+    if not (dimensions == 1 or dimensions == 2):
```
Comment (jokva) [inline camille/processors/fft.py]: Isn't this given from shape of dataframe?

File: camille/processors/fft.py [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+        raise ValueError("Cannot perform {}-dimensinal FFT on Pandas DataFrames"
+                         .format(dimensions))
+
+    funcname_dims = ('n' if dimensions > 2 else
+                     '2' if dimensions == 2 else '')
+    funcname_inverse = 'i' if inverse else ''
+    funcname = funcname_inverse + 'fft' + funcname_dims
```
Comment (jokva) [inline camille/processors/fft.py]: Format strings please.

**Pull request merged**